apps/src/applab/applab.js

```diff
@@ -80,6 +80,21 @@ export default Applab;
 var jsInterpreterLogger = null;
 
 /**
+ * Whether Maker Toolkit is enabled on this level.  Will be initialized on
+ * level load.
+ * @private {boolean}
+ */
+let makerEnabled = false;
```

Comment:
I've addressed this by giving maker its own redux module [in this subset of commits](https://github.com/code-dot-org/code-dot-org/compare/1bd049ce825a587c0aeb73e1da885af30e7330cb...3b373edcc59d9affbab1496a09a3d28755a5fac6).  This might seem like overkill now, but I'm about to start implementing a bunch of new UI revolving around maker so we might as well get the plumbing out of the way now.  I roughly modeled the new maker module on your jsDebugger module.  I haven't finished addressing all feedback in this review, but PTAL regarding this (fairly significant) part?